command/auth_enable.go

```diff
@@ -199,6 +200,13 @@ func (c *AuthEnableCommand) Flags() *FlagSets {
 		Usage:   "Select the version of the auth method to run. Not supported by all auth methods.",
 	})
 
+	f.StringSliceVar(&StringSliceVar{
+		Name:   flagNameAllowedManagedKeys,
+		Target: &c.flagAllowedManagedKeys,
+		Usage: "Comma-separated string or list of managed key registry entry names" +
```

Comment:
I believe this documentation and the test below are conflicting?

The test below is taking the comma separated value as a single list element, while my reading of this indicates there should have been two elements within the list.

Reply:
Updated the description for allowed-managed-keys. Will make changes for other flags in a different PR